### PR TITLE
[RJA-640] Project created when and by whom

### PR DESCRIPTION
**Pull request by azhuchkov**

### CODE REVIEW

File: ui/src/main/resources/assets/css/style.css [hunk 1]
```diff
@@ -302,6 +302,9 @@ table th{
     max-width: 80px;
 }
 
+#prj-list td.time {
```
Comment (bugzmanov) [inline ui/src/main/resources/assets/css/style.css]: what for?

Comment (azhuchkov) [inline ui/src/main/resources/assets/css/style.css]: it's a stub. actually I don't know what CSS rules should be there - looks
OK for me.

On Tue, Aug 21, 2012 at 6:18 PM, bugzmanov notifications@github.com wrote:

> In ui/src/main/resources/assets/css/style.css:
> 
> > @@ -302,6 +302,9 @@ table th{
> >      max-width: 80px;
> >  }
> > 
> > +#prj-list td.time {
> 
> what for?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/griddynamics/OpenGenesis/pull/445/files#r1424631.

## 

Sincerely yours
Andrey Zhuchkov
Software Engineer, Grid Dynamics
+7 927 107 7024 cell
Skype: zhuchkov.andrey
www.griddynamics.com
azhuchkov@griddynamics.com

Comment (bugzmanov) [inline ui/src/main/resources/assets/css/style.css]: And who suppose to fill this stub?

Comment (bugzmanov) [inline ui/src/main/resources/assets/css/style.css]: I thinkg empty rules should be removed. This css file already looks like peace of crap, please dont' pollute it without a reason

**Pull request merged**